yarn.lock

```diff
@@ -2248,43 +2248,51 @@
   optionalDependencies:
     npmlog "2 || ^3.1.0 || ^4.0.0"
 
-"@react-native-community/cli-clean@^11.0.0-alpha.2":
-  version "11.0.0-alpha.2"
-  resolved "https://registry.yarnpkg.com/@react-native-community/cli-clean/-/cli-clean-11.0.0-alpha.2.tgz#31e9a896df86dd11de06c1daced00142126c8ec0"
-  integrity sha512-IdZj/2CZYME/iWY4cAxKBWAr/pQTfj5I2NZp8Z4+u6/KrjP2PuoMkUcb+ONvwQCLpd9R62EnuaPzXVomH43d8A==
+"@react-native-community/cli-clean@^11.0.0-alpha.0":
+  version "11.0.0"
+  resolved "http://localhost:4873/@react-native-community%2fcli-clean/-/cli-clean-11.0.0.tgz#7225f8df011893de1cb740a0cad3dd2670574da5"
```

Comment:
@cipolleschi - sorry, just spotted these `localhost` URLs in the lockfile.

That might be OK for CI if that's the Verdaccio host, but won't work for contributors. Not sure if this is related to `alpha-0` being unpublished?

Reply:
> Working on a follow-up PR to fix these now.

https://github.com/facebook/react-native/pull/36661